### DISCUSSION TITLE
Removing en-US locale from bundle

### DISCUSSION THREAD

**Discussion by danilstoyanov** [js-fns/date-fns#1853] [General] [answered]
Hello guys!
Thank you all for contributing in such a good date lib.

But I wonder, is it possible to remove en-US locale from bundle if I need to use only Ru locale? I tried to use `webpack.ContextReplacementPlugin` which is mentioned in docs, but still en-US locale is getting to the bundle.

Or en-US locale is default and it's not possible to remove it somehow?

![image](https://user-images.githubusercontent.com/18240896/85258211-bf2d2d80-b46f-11ea-858f-1bf642772010.png)

**Comment by kossnocorp** [comment]
Hey! Yes, it's possible to replace the default locale. Could you please show code snippet with the usage of `webpack.ContextReplacementPlugin`?

**Reply by danilstoyanov** [reply]
Sure!
Here it is
![image](https://user-images.githubusercontent.com/18240896/85259027-00720d00-b471-11ea-84df-02d3a44da7f6.png)

**Comment by kossnocorp** [comment, marked as answer]
To be honest, I'm not quite sure how `webpack.ContextReplacementPlugin` works in this context, but to replace the default locale you can use `NormalModuleReplacementPlugin`:

```js
new webpack.NormalModuleReplacementPlugin(
  /locale\/en-US/,
  'date-fns/locale/ru'
)
```

It will replace `en-US` locale, so you can skip passing `ru` to functions. I just tested it on my app and works as expected:

![CleanShot 2020-06-22 at 11 37 15](https://user-images.githubusercontent.com/52201/85261140-d0c90200-b47c-11ea-85cf-ca06717cfb85.png)

**Reply by danilstoyanov** [reply]
Yeah, it's working good
Thank you for fast response!
